frontend/src/app/hierarchy-viewer/deleteHierarchy.dialog.html

```diff
@@ -0,0 +1,10 @@
+<h1 mat-dialog-title>
+    Delete {{data.name}}{{!data.name.includes('Hierarchy') ? ' Hierarchy' : null}}
+</h1>
+<mat-dialog-content>
+    <p>Are you sure you want to delete this hierarchy?</p>
```

Comment:
Then you can replace "this hierarchy" with the name of the hierarchy like you did above.